File: fourth/MayorDeTres.py
'''
Crea un programa que pida al usuario 3 números
y devuelva el mayor de los 3
 
 @author: Gonzalo Pulleiro
 
'''
print("Por favor, introduce 3 números")
num1 = int(input(""))
num2 = int(input(""))
num3 = int(input(""))

if(num1 > num2 and num1 > num3):
    print("{} es el número mayor.".format(num1))
elif (num1 < num2 and num2 > num3):
    print("{} es el número mayor.".format(num2))
else :
    print("{} es el número mayor.".format(num3))

File: Sencillos/fourth/MayorDeTres.java
package fourth;

/*
 * Crea un programa que pida al usuario 3 números
 * y devuelva el mayor de los 3
 * 
 *  @author: Gonzalo Pulleiro
 */

 import java.util.Scanner;

public class MayorDeTres {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.println("Por favor, introduce 3 números");
        int n1 = sc.nextInt();
        int n2 = sc.nextInt();
        int n3 = sc.nextInt();

        sc.close();

        if(n1 > n2 && n1 > n3){
            System.out.println(n1 + " es el número mayor.");
        } else if(n1 < n2 && n2 > n3){
            System.out.println(n2 + " es el número mayor.");
        } else {
            System.out.println(n3 + " es el número mayor.");
        }
    }
}
